include/sot/core/exp-moving-avg.hh

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2018,
+ * Julian Viereck
+ *
+ * CNRS/AIST
+ *
+ * This file is part of sot-core.
+ * sot-core is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version.
+ * sot-core is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.  You should
+ * have received a copy of the GNU Lesser General Public License along
+ * with sot-core.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef __SOT_EXPMOVINGAVG_H__
+#define __SOT_EXPMOVINGAVG_H__
+
+/* --------------------------------------------------------------------- */
+/* --- INCLUDE --------------------------------------------------------- */
+/* --------------------------------------------------------------------- */
+
+#include <dynamic-graph/entity.h>
+#include <dynamic-graph/signal-ptr.h>
+#include <dynamic-graph/signal-time-dependent.h>
+
+namespace dg = ::dynamicgraph;
+
+namespace dynamicgraph {
+  namespace sot {
+
+/* --------------------------------------------------------------------- */
+/* --- API ------------------------------------------------------------- */
+/* --------------------------------------------------------------------- */
+
+#if defined (WIN32)
+#  if defined (reader_EXPORTS)
+#    define SOTEXPMOVINGAVG_EXPORT __declspec(dllexport)
+#  else
+#    define SOTEXPMOVINGAVG_EXPORT __declspec(dllimport)
+#  endif
+#else
+#  define SOTEXPMOVINGAVG_EXPORT
+#endif
```

Comment:
Is this `define` different from SOT_CORE_DLLAPI in sot/core/config.hh (generated by CMake) ?